### PR TITLE
LPS-182122 Apply permission logic in users in Subtask

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
ci:test:sf

**Review by EugenioAntunes — APPROVED** [review]
Nice Job, Man!

**Review by Elvinho — APPROVED** [review]
LGTM ⚡

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/index.tsx [hunk 1]
```diff
@@ -43,7 +43,11 @@ type OutletContext = {
 		mergedSubtaskNames: string;
 		splitSubtaskNames: string;
 		subtaskIssues: TestraySubTaskIssue[];
-		testraySubtask: TestraySubTask;
+		testraySubtask: TestraySubTask & {
+			actions: {
+				[key: string]: string;
+			};
+		};
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/index.tsx:50]: @rafaelfs98 add in the type `TestraySubtask` this type: `actions: ObjectActionsItems`

See this example https://github.com/pt-liferay-solutions/liferay-portal/blob/3e876b037049874d545a8184acfb3b3461d6c59c/modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/types.ts#L161

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx [hunk 2]
```diff
@@ -54,7 +54,11 @@ import TaskHeaderActions from './TaskHeaderActions';
 
 type OutletContext = {
 	data: {
-		testrayTask: TestrayTask;
+		testrayTask: TestrayTask & {
+			actions: {
+				[key: string]: string;
+			};
+		};
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowTasks.tsx:61]: @rafaelfs98 add in the type `TestrayTask` this type: `actions: ObjectActionsItems`

See this example https://github.com/pt-liferay-solutions/liferay-portal/blob/3e876b037049874d545a8184acfb3b3461d6c59c/modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/types.ts#L161

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

@rafaelfs98 really good job on that man, thank you so much, I know it was a lot of things to test, 

you rock!

**Comment by kevenleone** [comment]
The points that I have mentioned can be develop in another moment, in your next PR add things change, thanks!

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**